docs/source/index.rst

```diff
@@ -290,6 +290,8 @@ Flax), PyTorch, and/or TensorFlow.
 +-----------------------------+----------------+----------------+-----------------+--------------------+--------------+
 |            DeiT             |       ❌       |       ❌       |       ✅        |         ❌         |      ❌      |
 +-----------------------------+----------------+----------------+-----------------+--------------------+--------------+
+|            Detr             |       ❌       |       ❌       |       ✅        |         ❌         |      ❌      |
```

Comment:
Don't forget to add it in the main README too :-)

Reply:
Done